### PR TITLE
gh #7 updating the dsHost test specification and the code

### PR DESCRIPTION
**Pull request by Anbukannadhasan**
Review request for the test spec document

### CODE REVIEW

File: docs/pages/dsHost_TestSpecification.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+|1|[Retrieve CPU Temperature](#retrieve-cpu-temperature)|Test if the module correctly retrieves the `CPU` temperature.|
+|2|[Obtain SOC ID](#obtain-soc-id)|Test if the module successfully obtains the 8-byte `SoC` ID programmed to the CHIP One Time Programmable area.|
+|3|[Fetch Host EDID](#fetch-host-edid)|Test if the module fetches the host `EDID` along with its length|
+-----------
+
+## Retrieve CPU Temperature
+
+|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----------|--------|--|--|--------------------------|
+|Invoke the module to retrieve `CPU` temperature, verify the returned temperature falls within valid operating range and ensure the `CPU` is in a normal operating state.|dsGetCPUTemperature|Y|N|N|
```
Comment (bhanucbp) [inline docs/pages/dsHost_TestSpecification.md:59]: for L3 check if we can use heat/cold chambers and retrieve the temperature information

File: docs/pages/dsHost_TestSpecification.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+### Control Plane Requirement - Retrieve CPU Temperature
+
+None
+
+## Obtain SoC ID
+
+|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----------|--------|--|--|--------------------------|
+|Trigger the module to obtain `SoC` ID and verify the returned ID matches the expected format and length.|dsGetSocIDFromSDK|Y|N|N|
```
Comment (bhanucbp) [inline docs/pages/dsHost_TestSpecification.md:77]: we can add the SoC ID in configuration file and compare with that

File: docs/pages/dsHost_TestSpecification.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+### Control Plane Requirement-Obtain SoC ID
+
+None
+
+## Fetch Host EDID
+
+|Description|HAL APIs|L2|L3|Control plane requirements|
+|-----------|--------|--|--|--------------------------|
+|Call the module function dsGetHostEDID() to fetch the host `EDID`. Verify the retrieved `EDID` data corresponds to the capabilities and specifications of the host device. Each device will have it's own EDID that is supports. I.E. Two pioneer tvs with the same resolution and size will have the same EDID values, but if those two pioneer tvs are of different sizes their EDID values will be different.|dsGetHostEDID|Y|N|N|
```
Comment (bhanucbp) [inline docs/pages/dsHost_TestSpecification.md:95]: remove the references to pioneer tvs and rephrase the description
Also mention the entries(key values) to be extracted from configuration file

**Review by bhanucbp — COMMENTED** [review]
address review comments

File: docs/pages/dsHost_TestSpecification.md (hunk 1 above)
Comment (hgfell683) [reply docs/pages/dsHost_TestSpecification.md:59]: Updated, please review the new test case.

File: docs/pages/dsHost_TestSpecification.md (hunk 2 above)
Comment (hgfell683) [reply docs/pages/dsHost_TestSpecification.md]: Added the reference to the configuration file to compare against.

File: profiles/source/Source_HostSettings.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
+Device:
```
Comment (Ulrond) [inline profiles/source/Source_HostSettings.yaml:1]: format is not correct, please use yaml linter.

There is no such thing as a combined setting.

Your either a dsAudio or a dsHdmiIn or 

your not all these at all times.


file: dsAudioConfiguration.yaml

```yaml
dsAudio:
 type: sink
 name: element
  features:
    extendedEnumsSupported: false
```

**Review by Ulrond — CHANGES_REQUESTED** [review]
discussion required on formatting, and seperation.

**Review by hari22yuva — APPROVED** [review]
Approved

**Pull request merged**